src/material-examples/toolbar-multirow/toolbar-multirow-example.html

```diff
@@ -7,12 +7,15 @@
     <span>Second Line</span>
     <span class="example-spacer"></span>
     <mat-icon class="example-icon">verified_user</mat-icon>
+    <span class="cdk-visually-hidden">Example user verified icon</span>
   </mat-toolbar-row>
 
   <mat-toolbar-row>
     <span>Third Line</span>
     <span class="example-spacer"></span>
     <mat-icon class="example-icon">favorite</mat-icon>
+    <span class="cdk-visually-hidden">Example heart icon</span>
     <mat-icon class="example-icon">delete</mat-icon>
+    <span class="cdk-visually-hidden">Example delete icon</span>
```

Comment:
These labels should be set as an `aria-label` on the `mat-icon`, rather than a `span` next to it.

Reply:
Changed as per your suggestion. Can you please check now